src/components/grid/GridComponent.vue

```diff
@@ -116,7 +124,7 @@ export default Vue.extend({
      */
     selected: function () {
       const selected = { all: true, row: [], col: [] }
-      if (this.grid === null) return selected
+      if (!this.grid.length) return selected
```

Comment:
suggest to always use if () {} to avoid errors and make code clearer

Reply:
Fixed!